core/src/main/scala/cats/Foldable.scala

```diff
@@ -595,6 +595,12 @@ trait Foldable[F[_]] extends UnorderedFoldable[F] with FoldableNFunctions[F] { s
       }
     }.value
 
+  /**
+   * Variant on travarse_ which enforces the "action or effect" expectation.
+   */
+  def traverseEach[G[_], A](fa: F[A])(f: A => G[Unit])(implicit G: Applicative[G]): G[Unit] =
```

Comment:
I personally don't love these aliases where two functions are actually always exactly the same.

Reply:
Hence why I think we should just replace `traverse_` in place, since it's binary compatible.